src/js/VirtualFC.js

```diff
@@ -220,10 +222,11 @@ const VirtualFC = {
             "BEEPER MUTE",
             "READY",
             "LAP TIMER RESET",
+            "CHIRP",
         ];
         FC.AUX_CONFIG_IDS = [
-            0, 1, 2, 4, 5, 6, 7, 8, 12, 13, 15, 17, 19, 20, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37,
-            39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54,
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 11, 12, 13, 15, 17, 19, 20, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35,
+            36, 37, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55,
```

Comment:
The highlighted selection shows a change in the array FC.AUX_CONFIG_IDS where the values have been modified and extended. The new array includes additional numbers and rearranges some existing numbers. Specifically, the numbers 3, 11, and 55 have been added, and the array's structure has been adjusted accordingly.

Reply:
i think es-lint re-arranged the 36, 37 to the next line automatically.

Reply:
What does 55 represent ?
Was looking at boxId_e enum (rc_modes.h)

Reply:
https://github.com/betaflight/betaflight/blob/caa8935e61f44d696d30676760ac98dae290c644/src/main/msp/msp_box.c#L104